thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -737,6 +765,7 @@ public Response initiateAlertFilterAutoTune(@PathParam("functionId") long id,
   public Response evaluateAlertFilterByFunctionId(@PathParam("functionId") long id,
       @QueryParam("start") String startTimeIso,
```

Comment:
Can start and end time be null? If not, please add @NotNull decorator.

Reply:
fixed. thanks